packages/schematics/angular/ng-new/index.ts

```diff
@@ -40,7 +40,6 @@ export default function (options: NgNewOptions): Rule {
   const workspaceOptions: WorkspaceOptions = {
     name: options.name,
     version: options.version,
-    experimentalAngularNext: options.experimentalIvy,
```

Comment:
This line is the only part that can be safely removed without causing a breaking change.  Even though the `experimentalAngularNext` is hidden it can still be used.

Reply:
Should we deprecate it? Or leave it as is?

Reply:
@hansl thoughts on the `experimentalAngularNext` option?